app/controllers/users_controller.rb

```diff
@@ -0,0 +1,72 @@
+class UsersController < ApplicationController
+  before_action :logged_in_user, only: [:index, :edit, :update, :destroy]
+  before_action :correct_user,   only: [:edit, :update]
+  before_action :admin_user,     only: :destroy
+
+  def index
+    @users = User.where(activated: true).paginate(page: params[:page], :per_page => 10)
+  end
+
+  def show
+    @user = User.find(params[:id])
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      @user.send_activation_email
+      flash[:info] = "Please check your email to activate your account."
```

Comment:
Would be nice if you would extract all messages into locales/en.yml and use them with `I18n.t()` helper.

Example:
```
en:  
   flash:
       email_activation: "Please check your email to activate your account."
```
```
flash[:info] = I18n.t('flash.email_activation')
```